scripts/pi-hole/php/groups.php

```diff
@@ -443,7 +443,7 @@ function JSON_error($message = null)
         $domains = explode(' ', trim($_POST['domain']));
         $total = count($domains);
         $added = 0;
-        $stmt = $db->prepare('INSERT INTO domainlist (domain,type,comment) VALUES (:domain,:type,:comment)');
+        $stmt = $db->prepare('REPLACE INTO domainlist (domain,type,comment) VALUES (:domain,:type,:comment)');
```

Comment:
```suggestion
        $stmt = $db->prepare('INSERT OR IGNORE INTO domainlist (domain,type,comment) VALUES (:domain,:type,:comment)');
```

I'd rather have it `IGNORE` if already existing. When `REPLACE`ing, this also forces a new `date_added`.

However, we'll have also have to change the way we count `INSERT`ed lines. I can push another commit addressing this if you don't mind.

Reply:
INSERT OR IGNORE has a confusing effect. You get a successfully added message, but it is not...

![image](https://user-images.githubusercontent.com/1998970/81867646-f226fc00-9568-11ea-9e35-280de3b1e99e.png)

Reply:
Why was this not added?